src/components/Button/Button.module.css

```diff
@@ -263,23 +263,21 @@
  * Disabled
  */
 .button--disabled {
+  pointer-events: none;
```

Comment:
We disable all the pointer events for these buttons to make them truly disabled (and this avoids having to add styles to override the `:hover`, `:active` and other pseudo-selectors